drivers/census_tracking/app/models/census_tracking/worksheet.rb

```diff
@@ -0,0 +1,90 @@
+        'Unaccompanied Trans Females Under Age 18' => she_t[:presented_as_individual].eq(true).and(shs_t[:age].lt(18)).and(c_t[:Gender].eq(2)),
+        'Unaccompanied Gender Non-conforming Under Age 18' => she_t[:presented_as_individual].eq(true).and(shs_t[:age].lt(18)).and(c_t[:Gender].eq(4)),
+        'Single Adult Males Age 18-24' => she_t[:presented_as_individual].eq(true).and(shs_t[:age].between(18..24)).and(c_t[:Gender].eq(1)),
+        'Single Adult Trans Males Age 18-24' => she_t[:presented_as_individual].eq(true).and(shs_t[:age].between(18..24)).and(c_t[:Gender].eq(3)),
+        'Single Adult Females Age 18-24' => she_t[:presented_as_individual].eq(true).and(shs_t[:age].between(18..24)).and(c_t[:Gender].eq(0)),
+        'Single Adult Trans Females Age 18-24' => she_t[:presented_as_individual].eq(true).and(shs_t[:age].between(18..24)).and(c_t[:Gender].eq(2)),
+        'Single Adult Gender Non-conforming Age 18-24' => she_t[:presented_as_individual].eq(true).and(shs_t[:age].between(18..24)).and(c_t[:Gender].eq(4)),
+        'Single Adult Males Age 25+' => she_t[:presented_as_individual].eq(true).and(shs_t[:age].gteq(25)).and(c_t[:Gender].eq(1)),
+        'Single Adult Trans Males Age 25+' => she_t[:presented_as_individual].eq(true).and(shs_t[:age].gteq(25)).and(c_t[:Gender].eq(3)),
+        'Single Adult Females Age 25+' => she_t[:presented_as_individual].eq(true).and(shs_t[:age].gteq(25)).and(c_t[:Gender].eq(0)),
+        'Single Adult Trans Females Age 25+' => she_t[:presented_as_individual].eq(true).and(shs_t[:age].gteq(25)).and(c_t[:Gender].eq(2)),
+        'Single Adult Gender Non-conforming Age 25+' => she_t[:presented_as_individual].eq(true).and(shs_t[:age].gteq(25)).and(c_t[:Gender].eq(4)),
+        'Number of households with at least one adult age 18+ and at least one child under age 18' => she_t[:head_of_household].eq(true).and(shs_t[:age].gteq(18)).and(she_t[:other_clients_under_18].gt(0)),
+        'Number of households with all members under age 18' => she_t[:head_of_household].eq(true).and(she_t[:children_only].eq(true)),
+        'Number of clients under age 18 in all households served' => she_t[:presented_as_individual].eq(false).and(shs_t[:age].lt(18)),
+        'Number of clients age 18-24 in all households served' => she_t[:presented_as_individual].eq(false).and(shs_t[:age].between(18..24)),
+        'Number of clients age 25+ in all households served' => she_t[:presented_as_individual].eq(false).and(shs_t[:age].gteq(25)),
+        'Total PIT (including clients of unknown gender)' => Arel.sql('1=1'),
+      }
+    end
+
+    private def service_scope
+      scope = GrdaWarehouse::ServiceHistoryEnrollment.
+        joins(:service_history_services, :client).
+        service_on_date(@filter.on).
+        in_project(@filter.effective_project_ids)
+
+      scope = scope.in_coc(coc_code: @filter.coc_codes) if @filter.coc_codes.present?
```

Comment:
Why not just use the FilterScope module for these? Then you can call things like `filter_for_race(scope)`.  Since you are starting with SHE, I think it will work.